### DISCUSSION TITLE
Wording in StreamPipes

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1172] [General]
Hi,
would just like to document the common wording

**Comment by flomickl** [comment]
So the wording in the StreamPipes Editor
![wording_pipeline_editor](https://user-images.githubusercontent.com/16739072/215322491-ace32cf4-647f-4061-b131-a444941929b5.png)


A: Overview of all Pipeline Elements
B: Data Sources / Streams
C: Processors
D: Sinks
E: Grouping (Pipeline Elements)
F: Search Bar
G: (Start) Tutorial
H: Save & Preview Section?
i: Zoom & Navigation?
J: Pipeline Assembly

something missing?

**Reply by bossenti** [reply]
@flomickl do you want to fill up the naming for every bounding box here or what is your intention? 🙂

**Reply by flomickl** [reply]
@bossenti  yep your intention is right ;)
I tried to describe some workflows and it was kind of hard to descripe.
Like See here... , Click on ..., and so on
but to get this structured it should be named all the time the same way and should be documented

**Reply by tenthe** [reply]
I added some terms that I mostly use, feel free to change them.

**Comment by flomickl** [comment]
Wording Pipeline Creation process
aka find the letter in the picture;)

![wording_pipeline_creation](https://user-images.githubusercontent.com/16739072/215323558-c20bf19b-8fe5-4d8d-be7b-5677155343fd.png)

A: Adatper 
B: Icon
C: Element Name 
D: Compatible Elements
E: Recommended Elements
F: Indicates if element is fully configured
G: Documentation
H: Delete Button
i: Connection
J: (is connected) (Input connection)
K: Edit Configurations
L: Processor
M: Sink
N: Pipeline Validation Hints
O: Modeling Error Descriptions
P: (not connected)
Q: Resulting Event Stream / Processing Element Output

**Reply by tenthe** [reply]
Same here, I added some suggestions. Feel free to change them if they are not clear.
